src/concepts/glean.md

```diff
@@ -0,0 +1,72 @@
+# Glean - mobile product analytics
+
+For Mozilla, getting reliable data from our products is critical to inform our decision making. Glean is our new product analytics & Telemetry solution that provides that data for our mobile products.
+It aims to be easy to integrate, reliable and transparent by providing an SDK and integrated tools.
+
+It currently supports Android products, while iOS support is planned.
+Note that this is different from Telemetry for Firefox Desktop ([1](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/index.html), [2](https://docs.telemetry.mozilla.org/concepts/choosing_a_dataset.html)), although it provides similar capabilities.
+
+Contents:
```

Comment:
This article is rather short. Do we need a ToC?

In any case `<!-- toc -->` should give that automatically.